static/webmanifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Neomad. The Open Digital-Nomad Community",
+  "short_name": "Neomad",
+  "description": "Find a local place to work, get connected to other remote workers.",
+  "background_color": "#279e6a",
+  "theme_color": "#279e6a",
+  "dir": "rtl",
```

Comment:
Are you sure about the right-to-left?

Reply:
Oh ....... I stopped coffee the morning ^^